xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -83,20 +83,21 @@ class CActiveAESink : private CThread
 {
 public:
   explicit CActiveAESink(CEvent *inMsgEvent);
-  void EnumerateSinkList(bool force);
+  void EnumerateSinkList(bool force, std::string driver);
   void EnumerateOutputDevices(AEDeviceList &devices, bool passthrough);
   void Start();
   void Dispose();
   AEDeviceType GetDeviceType(const std::string &device);
   bool HasPassthroughDevice();
   bool SupportsFormat(const std::string &device, AEAudioFormat &format);
+  bool DeviceExist(std::string driver, std::string device);
   CSinkControlProtocol m_controlPort;
   CSinkDataProtocol m_dataPort;
 
 protected:
   void Process() override;
   void StateMachine(int signal, Protocol *port, Message *msg);
-  void PrintSinks();
+  void PrintSinks(std::string &driver);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types